src/components/Company/CompanyName/CorrectNameOptions.vue

```diff
@@ -6,6 +6,7 @@
     <v-expansion-panels v-model="panel">
       <v-expansion-panel
         v-for="(item,i) in displayedOptions"
+        :id="item.id"
```

Comment:
Maybe add a name to this panel's id?

Reply:
For sure.
Something like `:id="x-panel: ${item.id}" ? Is that what you mean?

Reply:
Yes.